### ISSUE TITLE
Browsing previous dates

### ISSUE THREAD

**Issue by tylervick** [trotzig/react-available-times#38]
Is there any way to navigate to weeks prior to the current one? I don't see any way to configure this, nor why this functionality would be missing. 

Thanks!

**Comment by trotzig** [comment]
Nope, not right now. The main use-case for this component is to book a meeting, so to us it didn't make sense to be able to navigate back in time.
